content/beginner/190_secrets/consume-a-secret.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Access the Secret from a Pod"
+date: 2019-04-09T00:00:00-03:00
+weight: 11
+draft: false
+---
+
+#### Deploy a Pod to Consume the Secret
+Create a YAML file (podconsumingsecret.yaml) with the following pod definition:
```

Comment:
Can this be a here doc?

Reply:
I missed this one, @brentley. Can you give me some more details on what you are looking for?

Reply:
Found an example of what I believe you meant with the `cat << EOF` file creation. I updated both this file creation and the cluster config creation YAML to use that format. This happens to automatically substitute environment variables as well.